src/components/Loader/PeopleTable/PersonItem.tsx

```diff
@@ -0,0 +1,56 @@
+import { Link, useParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { Person } from '../../../types';
+
+export const PersonItem = ({ person }: { person: Person }) => {
+  const { slug } = useParams();
+
+  return (
+    <tr
+      data-cy="person"
+      className={classNames({
+        'has-background-warning': slug === person.slug,
+      })}
+    >
+      <td>
+        <Link
+          to={`/people/${person.slug}`}
+          className={classNames({
+            'has-text-danger': person.sex === 'f',
+          })}
+        >
+          {person.name}
+        </Link>
+      </td>
+
+      <td>{person.sex}</td>
+      <td>{person.born}</td>
+      <td>{person.died}</td>
+
+      <td>
+        {person.mother ? (
+          <Link
+            className="has-text-danger"
```

Comment:
The link to the person's mother uses a hardcoded class name 'has-text-danger' which is inconsistent with the conditional class applied to the person's name based on their sex (line 19). It's important to maintain consistency in styling. If the red text is meant to indicate a female, consider applying the same conditional logic as used for the person's name.